web/src/actions/activities.js

```diff
@@ -76,79 +70,6 @@ export const addActivityContractor = (key, { save = saveApd } = {}) => async (
   dispatch(updateActivity(key, updates));
 };
 
-export const uploadActivityContractorFile = (
```

Comment:
Guessing these aren't actions we need to take any longer?

Reply:
Indeed, at least not for now. We used to support uploading arbitrary attachments for contractor resources, but we took it out because it didn't work right. Rather than just leave it all in there, I yoinked it. "YAGNI" as @abrouilette says, and when we do eventually need this functionality again, we can look back at this history to decide if we want to pull it up or do it differently.